src/installer.ts

```diff
@@ -8,6 +8,7 @@ import fs from 'fs';
 import os from 'os';
 
 type InstallationType = 'dist' | 'manifest';
+export type StableAliasType = 'stable' | 'oldstable';
```

Comment:
I think it is better to create enums or use a similar approach as for [setup-dotnet](https://github.com/actions/setup-dotnet/blob/main/src/setup-dotnet.ts#L8-L16).